app/controllers/providers_controller.rb

```diff
@@ -4,9 +4,11 @@ class ProvidersController < ApplicationController
   def index
     @providers = Provider.all
     render_manage_ui if @providers.empty?
+    redirect_to controller: 'providers', action: 'show', code: @providers.first.institution_code if @providers.size == 1
```

Comment:
`redirect_to provider_path(@providers.first) if @providers.size == 1` ?

Reply:
Might also be able to:

```
redirect_to @providers.first if @providers.size == 1
```

Reply:
yes